backend/src/services/calendar/api/methods.py

```
@@ -0,0 +1,2990 @@
+    
+    # Get existing event (must be recurring)
+    event = get_event(session, calendar_id, event_id, user_id)
+    if event is None:
+        raise EventNotFoundError(event_id)
+    
+    # Parse query parameters with validation
+    max_results = parse_int_param(params, "maxResults", default=250, max_value=2500)
+    page_token = params.get("pageToken")
+    time_min = params.get("timeMin")
+    time_max = params.get("timeMax")
+    time_zone = params.get("timeZone")
```

Ruff (0.14.13)

Local variable `time_zone` is assigned to but never used

Remove assignment to unused variable `time_zone`

(F841)